CHANGELOG.rst

```diff
@@ -10,6 +10,35 @@ Unreleased changes are documented in files in the `changelog.d`_ directory.
 
 ..  scriv-insert-here
 
+.. _changelog-0.21.0:
+
+0.20.1 — 2024-12-04
```

Comment:
What's all this now? `0.21.0` and `0.20.1` are different.

Reply:
Oh, rats! I was going to mark it as 0.21.0 and then switched to 0.20.1 and _apparently_ didn't catch them all.

Reply:
Amended! :shushing_face: 

Thanks for the catch.